src/java.base/share/classes/sun/security/ec/ECPublicKeyImpl.java

```diff
@@ -40,8 +34,6 @@
 import java.security.interfaces.*;
 import java.security.spec.*;
 
-import jdk.crypto.jniprovider.NativeCrypto;
```

Comment:
The import of `BigInteger` is not present upstream, nor is the final newline; compare with c86650f47c04e5d349dc22017fa14b94f93ad6df.

Reply:
Sorry about that. I was looking at the wrong version, butI think I got everything now.